lib/src/sql/changefeed.rs

```diff
@@ -0,0 +1,69 @@
+use crate::sql::comment::shouldbespace;
+use crate::sql::error::IResult;
+use nom::bytes::complete::tag_no_case;
+use serde::{Deserialize, Serialize};
+use std::fmt::{self, Display, Formatter};
+use std::str;
+use std::time;
+
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize, Hash)]
+pub struct ChangeFeed {
+	pub expiry: time::Duration,
+}
+
+impl ChangeFeed {
+	pub fn enabled(d: time::Duration) -> Self {
+		ChangeFeed {
+			expiry: d,
+		}
+	}
+}
+
+impl Display for ChangeFeed {
+	fn fmt(&self, f: &mut Formatter) -> fmt::Result {
+		write!(f, "CHANGEFEED {}", crate::sql::duration::Duration(self.expiry))?;
```

Comment:
Could we import this at the top of the file?

```rs
use crate::sql::duration::{duration, Duration};
```

```rs
write!(f, "CHANGEFEED {}",Duration(self.expiry))?;
```

Reply:
Sure! [92e49b4](https://github.com/surrealdb/surrealdb/pull/2189/commits/92e49b460254dc694683ee6fc9bb005679fcf0e9)